### ISSUE TITLE
⛔ ♿ 🚀 Feature Request: UI improvement - Warning/error messages "Learn More"

### ISSUE THREAD

**Issue by zeinnicholas** [elementor/elementor#8197]
## Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
The "Learn More" button on warning/error messages needs to be more prominent, since users keep going to FB groups to ask for help that could easily be answered by clicking on the button, but in this case, the "Cancel/Go Back" button has more contrast, leading them to click it out instead of following up to learn more. The way it looks right now, it seems the "Learn More" button is deactivated (can't click) because it's too faded.

**Describe the solution you'd like**
Maybe changing the background color to green (or other positive color) and the text to white, or using the Elementor purple color, would encourage users to click the button instead of dismissing the notice.

![61303825_2239153882829320_5516241839323086848_n](https://user-images.githubusercontent.com/26891676/58373118-4b80c000-7eff-11e9-9e16-45d88c88a2ca.jpg)

**Describe alternatives you've considered**
None atm.

**Additional context**
Lots and lots of posts on FB groups asking for help for problems that could be easily addressed by the documentation that these buttons lead to.

**Comment by nicholaszein** [comment]
Hey buddy! This issue is no longer relevant since the UI has changed and now buttons are more discernible. 

:)
